contracts/interfaces/IERC20Detailed.sol

```diff
@@ -0,0 +1,13 @@
+//SPDX-License-Identifier: MIT
```

Comment:
Are these interfaces necessary? Is this in preparation for using a proxy or making them upgradable? Seems like a bit of overkill to do this now IMO.

Reply:
It is not preparation for proxy. I created interfaces because we are using contract-to-contract calls. Without interface you need to deploy full contract code just to be able to do a call. With interface you deploy few lines of interface code.
So in this case it is not overkill but good practise.  
It might be overkill if interfaces were in use only for its main contract but it's not the case here.

Reply:
Hmm, I'm not sure i follow. Not sure you would need to deploy a full contract to do a call. You can just do: `ValidatorRegistry(address)` and then just call methods on it.

We can leave as-is since i don't think it hurts and is good for the future.

Reply:
hmm actually you right, it does not influence the gas while deploying (for some reason I thought it does) but still it's a good practice when doing contract-to-contract calls like this: `ValidatorRegistry(address)`.

Reply:
Cool sounds good to me.